filer/admin/clipboardadmin.py

```diff
@@ -75,16 +74,17 @@ def ajax_upload(request, folder_id=None):
     Receives an upload from the uploader. Receives only one file at a time.
     """
     mimetype = "application/json" if request.is_ajax() else "text/html"
-    content_type_key = 'mimetype' if LTE_DJANGO_1_4 else 'content_type'
-    response_params = {content_type_key: mimetype}
+    response_params = {'content_type': mimetype}
     folder = None
     if folder_id:
         try:
             # Get folder
             folder = Folder.objects.get(pk=folder_id)
         except Folder.DoesNotExist:
-            return HttpResponse(json.dumps({'error': NO_FOLDER_ERROR}),
-                                **response_params)
+            return HttpResponse(
```

Comment:
Good point. I wonder if there even is any case where it would make sense to return json as ``text/html``.